components/motor/ulnstepper/28byj-48.go

```diff
@@ -264,11 +277,7 @@ func (m *uln28byj) GoFor(ctx context.Context, rpm, revolutions float64, extra ma
 	m.targetStepPosition, m.stepperDelay = m.goMath(rpm, revolutions)
 	m.lock.Unlock()
 
-	err = m.doRun(ctx)
-	if err != nil {
-		return errors.Errorf(" error while running motor %v", err)
-	}
-	return nil
+	return m.opMgr.WaitTillNotPowered(ctx, 10*time.Millisecond, m, m.Stop)
```

Comment:
Let's use WaitUntilSuccess instead, I think wait until not powered is way too specific a function and I would like to remove it.